theme/breadcrumbs.html

```diff
@@ -12,22 +12,30 @@
     {% endif %}
     {% if page %}<li>{{ page.title }}</li>{% endif %}
 
-
-    {% if page and page.file.src_path != "actions.md" 
-          and not "actions/" in page.file.src_path 
-          and page.file.src_path != "plugins/available-plugins.md" %}
+    {% if page and page.file.src_path != "actions.md" and page.file.src_path != "plugins/available-plugins.md" %}
       <li class="wy-breadcrumbs-aside">
         {%- block repo %}
-        {% if page and page.edit_url %}
-          <a href="{{ page.edit_url }}"
+
+        {% set icon_class_block %}
           {%- if config.repo_name|lower == 'github' %}
             class="icon icon-github"
           {%- elif config.repo_name|lower == 'bitbucket' %}
             class="icon icon-bitbucket"
           {%- elif config.repo_name|lower == 'gitlab' %}
             class="icon icon-gitlab"
-          {% endif %}> Edit on {{ config.repo_name }}</a>
-        {% endif %}
+          {% endif %}
+        {% endset %}
+
+        {% set edit_docs_file_url %}
+          {%- if "actions/" in page.file.src_path %}
+            {{ config.extra.fastlane_repo_base_edit_uri }}/{{ page.file.src_path|string|safe|replace(".md", ".rb") }}
+          {%- elif page.edit_url %}
+            {{ page.edit_url }}
+          {%- endif %}
+        {% endset %}
+
+        <a href="{{ edit_docs_file_url }}" {{ icon_class_block }}>Edit on {{ config.repo_name }}</a>
```

Comment:
I'm struggling with the whitespace around all the `% set` blocks… What's the best way around here? For source legibility this is okay. However it produces slightly wild output (including whitespace in the actual anchor link… that luckily gets stripped in all the UAs I've tried… but is not really the intended output I guess):

<img width="1940" alt="Screen Shot 2021-11-21 at 1 04 41" src="https://user-images.githubusercontent.com/1784648/142745515-57897d98-5345-4a33-8074-5ca19e858132.png">

Would it be OK to somehow inline the code blocks a little more, so the template parser doesn't inject all the whitespace into the variable content?